packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -467,8 +467,8 @@ const Dropdown = React.forwardRef(
             // NOTE: does not prevent click
             evt.preventDefault();
             // focus on the element as per readonly input behavior
-            if (mergedRef.current !== undefined) {
-              mergedRef.current.focus();
+            if (mergedRef !== undefined) {
+              mergedRef?.current?.focus();
             }
```

Comment:
```suggestion
            mergedRef?.current?.focus();
```

does optional chaining by itself accomplish the same result?

Reply:
Yes @emyarod .  I agree, the conditional if statement is unnecessary .  Thank you